spec/app/controllers/vulneruby_engine/static_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'capybara'
+require 'rails_helper'
+
+RSpec.describe('Static Controller', type: :request) do
+  describe 'Static content', type: :system do
+    # Set up teh Capybara & Selenium things
+    before do
+      Capybara.register_driver :headless_chromium do |app|
+        options = Selenium::WebDriver::Chrome::Options.new
+        options.add_argument("--headless")
+        options.add_argument("--no-sandbox")
+        options.add_argument("--disable-dev-shm-usage")
+        options.add_argument("--window-size=1280,900")
+        Capybara::Selenium::Driver.new(app, browser: :chrome, capabilities: options)
+      end
+      Capybara.javascript_driver = :headless_chromium
+    end
+
+    it 'does whatever this is' do
```

Comment:
🤨

```suggestion
    it 'retrieves the static page' do
```

Reply:
I am not done with the test, more I'm trying to make it work in the pipeline :D